manifests/overlays/authentication/secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dashboard-oauth-config
+type: Opaque
+data:
+  cookie_secret: VGtob01rbFVkRzFJUkdwNlkyTm1jSEZMZVdOQmR6MDk=
```

Comment:
It was harcoded and public before, not even encoded, we should definitely change that in the future, I didn't want to hold back this feature too much because of this.

Reply:
I've opened https://github.com/opendatahub-io/odh-dashboard/issues/337